crypto/mem_sec.c

```diff
@@ -31,6 +31,10 @@
 # include <unistd.h>
 # include <sys/types.h>
 # include <sys/mman.h>
+# if defined(OPENSSL_SYS_LINUX)
```

Comment:
Yes, I agree. In this case, I'd prefer to keep it using the same type of checks that are in the rest of the file, to keep it internally consistent.